css/main.css

```diff
@@ -73,15 +80,35 @@ ul {
     padding: 10px 15px;
     border: 1px solid #fefefe;
     border-radius: 3px;
-    
+
     font-size: 11pt;
-    
+
     background: transparent;
     color: #fefefe;
-    
+
     transition: background ease-in-out 0.1s;
 }
 
+.top-container > div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#tunilogo {
+    width: 23em;
+    height: 8em;
+    filter: brightness(100);
+    background-image: url(../img/tuni.png);
```

Comment:
Logo is not a background image

Reply:
Did it this way for easier implementation of hover effect